scripts/update/node.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 # Node update check
 # author: liara
+#shellcheck source=sources/functions/npm
 . /etc/swizzin/sources/functions/npm
-if [[ -f /etc/apt/sources.list.d/nodesource.listE ]]; then mv /etc/apt/sources.list.d/nodesource.listE /etc/apt/sources.list.d/nodesource.list; fi
```

Comment:
This was just one of those random one-off typos accidentally created by the use of `sed -iE`